examples/src/bunnymark/mod.rs

```diff
@@ -366,11 +380,51 @@ impl crate::framework::Example for Example {
 
     fn resize(
```

Comment:
I'm not sure if this is intentional, but unlike in https://github.com/gfx-rs/wgpu/issues/5890#issuecomment-2242094769, the bunnies do not get repositioned within the window's new vertical bounds.

So bunnies bouncing within a 150pt tall window will continue bouncing in that small 150pt vertical range even if the window is resized to 2000 points, for instance:


https://github.com/user-attachments/assets/75ec0d4a-2086-4eb1-83d0-c5f172896650

Reply:
The bunnies spawn correctly upon pressing spacebar, but the ones which already spawned bounce in this limited range

Reply:
I'm taking a looking this weekend, sorry have been really busy

Reply:
I think the jump speed is slow as it grows, try leaving it for a bit and it should jump higher. This is what i'm seeing on my side.

Reply:
I guess it depends on the intent of the sample. My expectation would be they would be repositioned relative to the new bounds of the canvas when the window is resized.

Either way is fine from my perspective 👍, I am not a wgpu contributor so ultimately your decision :)

Reply:
I'm new here as well so I'll defer it to @ErichDonGubler

Reply:
bro idk 😆💦

Gonna punt to the rest of @gfx-rs/wgpu for opinion on this. I won't, however, punt on merging this, because stale PRs are no fun, and this behavior doesn't really seem risky, just subject to opinion.

@gfx-rs/wgpu: I'm gonna merge this. If you have qualms about current behavior, then let's fix/discuss it in a separate PR.